Dockerfile

```diff
@@ -42,17 +41,10 @@ RUN source scl_source enable rh-python36 && \
     fix-permissions /opt/app-root && \
     rpm-file-permissions
 
-COPY ./.s2i/bin/ /usr/libexec/s2i
-
-ENV APPROOT="/opt/app-root/src/tensorflowapp-sample"  VERSION="1"
-COPY ["tensorflowapp-sample", "${APPROOT}"]
-COPY ["requirements.txt", "${APPROOT}"]
+COPY ./.s2i/bin/usage /tmp/scripts/
 
-WORKDIR $APPROOT
+RUN ls -la /tmp/scripts
 
+COPY ["requirements.txt", "/opt/app-root/src/"]
+RUN pip install --upgrade pip
 RUN pip install -r requirements.txt --default-timeout=100
-
-USER 1001
```

Comment:
please put lines 55-56 back in as they are best practices for this sort of thing. In other places we set user 1001 as owner of folders etc and without this we are not guaranteed to be the same user.

Reply:
Added
'''
USER 1001
ENTRYPOINT [ "/tmp/scripts/uid_entrypoint" ]
CMD ["/tmp/scripts/usage"]
'''